strings/0003/char-frequency-count_francine.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+#include <string.h>
+
+void countFrequency(char[]);
+
+int main()
+{
+    char array[100];
+
+    //get string
+    printf("Enter string: ");
+    gets(array);
+
+    //function call
+    countFrequency(array);
+
+    return 0;
+}
+
+void countFrequency(char array[])
+{
+    int length, i, j, count = 0, flag = 0;
+
+    //get length of array
+    length = strlen(array);
+
+    for (i = 0; i < length; i++) {
```

Comment:
I think the loop can be improved.

If you noticed, if you have a a string with duplicates, the print is also duplicated by x times a character appears.

Maybe you can use the `alreadyCompared` method? Or you can use another method to temporarily store the results somewhere then print them again after the loop.